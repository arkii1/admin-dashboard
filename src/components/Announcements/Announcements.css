.announcements {
  background-color: transparent;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  gap: 1rem;
  -ms-grid-column: 2;
  grid-column: 2;
  -ms-grid-row: 1;
  grid-row: 1;
  height: 100%;
}

.announcements__header {
  font-size: 2rem;
  font-weight: bold;
}

.announcements__items {
  height: 100%;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  gap: 1rem;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  background-color: #ffffff;
  border-radius: 10px;
  padding: 2rem;
  -webkit-box-shadow: 5px 5px 4px 2px rgba(0, 0, 0, 0.2);
          box-shadow: 5px 5px 4px 2px rgba(0, 0, 0, 0.2);
}

.announcements__item__header {
  font-size: 1.5rem;
  font-weight: 700;
}

.announcements__item__p {
  font-size: 1.2rem;
  color: #636363;
}

.announcements__br {
  background-color: #636363;
  width: 100%;
}
/*# sourceMappingURL=Announcements.css.map */