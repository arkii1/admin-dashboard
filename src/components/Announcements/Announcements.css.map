{
    "version": 3,
    "mappings": "AAEA,AAAA,cAAc,CAAC;EACX,gBAAgB,EAAE,WAAW;EAC7B,OAAO,EAAE,IAAI;EACb,cAAc,EAAE,MAAM;EACtB,GAAG,EAAE,IAAI;EACT,WAAW,EAAE,CAAC;EACd,QAAQ,EAAE,CAAC;EACX,MAAM,EAAE,IAAI;CACf;;AAED,AAAA,sBAAsB,CAAC;EACnB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;CACpB;;AAED,AAAA,qBAAqB,CAAC;EAClB,MAAM,EAAE,IAAI;EACZ,OAAO,EAAE,IAAI;EACb,cAAc,EAAE,MAAM;EACtB,GAAG,EAAE,IAAI;EACT,eAAe,EAAE,UAAU;EAC3B,gBAAgB,ECvBV,OAAO;EDwBb,aAAa,EAAE,IAAI;EACnB,OAAO,EAAE,IAAI;EACb,UAAU,EAAE,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,GAAG,CAAC,kBAAe;CAC9C;;AAED,AAAA,4BAA4B,CAAC;EACzB,SAAS,EAAE,MAAM;EACjB,WAAW,EAAE,GAAG;CACnB;;AAED,AAAA,uBAAuB,CAAC;EACpB,SAAS,EAAE,MAAM;EACjB,KAAK,ECjCK,OAAO;CDkCpB;;AAED,AAAA,kBAAkB,CAAC;EACf,gBAAgB,ECrCN,OAAO;EDsCjB,KAAK,EAAE,IAAI;CACd",
    "sources": [
        "Announcements.scss",
        "../../varaibles.scss"
    ],
    "names": [],
    "file": "Announcements.css"
}