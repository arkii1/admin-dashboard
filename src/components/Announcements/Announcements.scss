@import "../../varaibles.scss";

.announcements {
    background-color: transparent;
    display: flex;
    flex-direction: column;
    gap: 1rem;
    grid-column: 2;
    grid-row: 1;
    height: 100%;
}

.announcements__header {
    font-size: 2rem;
    font-weight: bold;
}

.announcements__items {
    height: 100%;
    display: flex;
    flex-direction: column;
    gap: 1rem;
    justify-content: flex-start;
    background-color: $c-white;
    border-radius: 10px;
    padding: 2rem;
    box-shadow: 5px 5px 4px 2px rgba(0,0,0,0.2);
}

.announcements__item__header {
    font-size: 1.5rem;
    font-weight: 700;
}

.announcements__item__p {
    font-size: 1.2rem;
    color: $c-dark-grey;
}

.announcements__br {
    background-color: $c-dark-grey;
    width: 100%;
}