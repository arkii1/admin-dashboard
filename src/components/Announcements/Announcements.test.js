import React from "react"
import ReactDOM from "react-dom/client"
import { cleanup } from "@testing-library/react"
import renderer from "react-test-renderer"
import Announcements from "./Announcements"

afterEach(cleanup)

it("renders without crashing", () => {
  const container = document.createElement("div")
  const root = ReactDOM.createRoot(container)
  root.render(<Announcements />)
})

it("matches snapshot", () => {
  const tree = renderer.create(<Announcements />).toJSON()
  expect(tree).toMatchSnapshot()
})
