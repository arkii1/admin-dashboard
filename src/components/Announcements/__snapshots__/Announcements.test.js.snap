// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="announcements"
>
  <h1
    className="announcements__header"
  >
    Announcements
  </h1>
  <div
    className="announcements__items"
  >
    <h2
      className="announcements__item__header"
    >
      Site Maintenance
    </h2>
    <p
      className="announcements__item__p"
    >
      Verstibulum condimentum tellus lacus, in accumsan elit maximus ac. Donec hendrerit sodales congue...
    </p>
    <hr
      className="announcements__br"
    />
    <h2
      className="announcements__item__header"
    >
      Community Share Day
    </h2>
    <p
      className="announcements__item__p"
    >
      Nam vel lectus tincidunt, rutrum nulla eu, ornare libero. Aliquam dictum accumsan porttitor...
    </p>
    <hr
      className="announcements__br"
    />
    <h2
      className="announcements__item__header"
    >
      Updated Privacy Policy
    </h2>
    <p
      className="announcements__item__p"
    >
      Phasellus efficitur nisi eget elit consectetur, eget condimentum ante auctor. Cras fringilla sagittis sem in mattis...
    </p>
  </div>
</div>
`;
